src/battle_message.c

```diff
@@ -3163,7 +3163,8 @@ static void GetBattlerNick(u32 battler, u8 *dst)
         }                                                               \
     }                                                                   \
     GetBattlerNick(battler, text);                                      \
-    toCpy = text;
+    toCpy = text;                                                       \
+    dstWidth = GetStringLineWidth(fontId, dst, letterSpacing, lineNum, dstSize, FALSE);
 
 #define HANDLE_NICKNAME_STRING_LOWERCASE(battler)                       \
```

Comment:
Would it be worth changing this to an inline instead? Macros that are just functions look a bit cursed

Reply:
eh, it's fine